cvat-ui/src/utils/filter-annotations.ts

```diff
@@ -32,9 +32,13 @@ export function filterAnnotations(annotations: ObjectState[], params: FilterAnno
         }
 
         // GT tracks are shown only on GT frames
-        if (workspace === Workspace.REVIEW && groundTruthJobFramesMeta && frame) {
+        if (workspace === Workspace.REVIEW && groundTruthJobFramesMeta?.includedFrames && frame) {
             if (state.objectType === ObjectType.TRACK && state.isGroundTruth) {
-                return groundTruthJobFramesMeta.includedFrames.includes(frame);
+                // includedFrames has absolute numeration of frames
+                // frame N actually corresponds to startFrame + N * frameStep in absolute numeration
+                const { frameStep, startFrame } = groundTruthJobFramesMeta;
+                const dataFrameNumber = frame * frameStep + startFrame;
```

Comment:
It seems like we already have operations with transformation of relative/absolute frames [here](https://github.com/cvat-ai/cvat/blob/develop/cvat-core/src/frames.ts#L329). Can we re-use it?

Also it seems they [are duplicated](https://github.com/cvat-ai/cvat/blob/develop/cvat-ui/src/components/quality-control/task-quality/allocation-table.tsx#L47) inside of allocation table. Seems we need to refactor it.

Reply:
Probably you meant `getDataFrameNumber` as `getDataStartFrame` has another expression

Reply:
Yes, but it feels like we dont need multiple expressions for the same thing. The math behind it is the same. Its better to use something like:
```
const dataFrameNumber = getDataFrameNumber(meta, frame);
```
instead of writhing the transformation formula each time.

Reply:
Applied changes